app/models/campaign_bundle.rb

```diff
@@ -105,11 +105,15 @@ def daily_impressions_by_audience
     end
   end
 
+  # TODO: rename date_range to something more appropriate for ui only concerns like: date_range_string
```

Comment:
Is there a name you want? We should just go ahead and change it instead of leaving todos or open an issue.

Reply:
Being lazy here because this is also defined in another place, but I'm not ready to make the refactor for everything yet.